commands/utility/wiki.js

```diff
@@ -0,0 +1,45 @@
+const Discord = require('discord.js')
+const got = require('got')
+
+exports.run = async (handler, message, args, pre) => {
+	if (!args.text) {
+		return message.reply('must provide a text argument.')
+	}
+
+	let query = `https://en.wikipedia.org/w/api.php?action=query&prop=pageimages|extracts&titles=${args.text}&exintro=&exsentences=2&redirects=&explaintext=&formatversion=2&piprop=original&format=json`
+
+	try {
+		let response = await got(query)
+		let wikiJson = JSON.parse(response.body).query.pages[0]
+
+		if (wikiJson.missing) {
+			return await message.reply('page not found!')
+		}
+
+		let embed = new Discord.MessageEmbed()
+			.setColor('BLUE')
+			.setTitle(wikiJson.title)
+			.setDescription(`${wikiJson.extract}\n\n[Permalink](https://en.wikipedia.org/wiki/${wikiJson.title.replace(/( )/gm, '_')})`)
```

Comment:
Not a big issue, but the capturing group and the "m" flag are both unnecessary here. Just `.replace(/ /g, '_')` would work fine.